tests/src/test/scala/slinky/core/annotations/ReactAnnotatedComponentTest.scala

```diff
@@ -213,6 +215,64 @@ object DerivedStateComponent {
   }
 }
 
+@react class SubComponentWithReactElementContainers extends StatelessComponent {
+  case class Props(seq: Seq[ReactElement],
+                   list: List[ReactElement],
+                   option: Option[ReactElement],
+                   jsUndefOr: js.UndefOr[String],
+                   attempt: Try[ReactElement],
```

Comment:
Could we drop this, `function`, and `future` since they aren't being tested?

Reply:
Actually they are being tested when we pass them from a Parent to Sub component via Props. Currently doesn't work.

I did originally have Future/Functions being supported in the div() but we decided to drop it since it meant evaluating them at that point.

Reply:
Ah, I understand now!